ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.html

```diff
@@ -715,7 +715,7 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
               </ng-container>
             </tab>
 
-          <tab *ngIf="showContactInformation" heading="Contact Information">
+          <tab *ngIf="showContactInformation && (!hasMrViewPermission && hasViewOnlyDSSDataPermission)" heading="Contact Information">
```

Comment:
Contact Information still visible for `hasMrViewPermission`, yes? 
If user has `hasMrViewPermission`: Contact Information should be visible